src/utils.jl

```diff
@@ -11,7 +11,12 @@ arbitrary axes.
 
 See also: [`imfilter`](@ref).
 """
-centered(A::AbstractArray) = OffsetArray(A, map(n->-((n+1)>>1), size(A)))
+function centered(A::AbstractArray)
+    all(isodd.(size(A))) || error("entries must be odd, got $A")
```

Comment:
This didn't used to be a requirement, though its behavior wasn't documented (which is not good):
```julia
julia> centered(1:4)
1:4 with indices -1:2
```

I think if you delete this check it's good to go. (Also for future purposes, that error message is confusing: "entries must be odd" sounds like the values of A, and that's compounded by the fact that you print A itself rather than its axes.)